pkg/cloudprovider/providers/gce/gce_loadbalancer_external.go

```diff
@@ -265,7 +265,7 @@ func (gce *GCECloud) ensureExternalLoadBalancer(clusterName, clusterID string, a
 
 	// Once we've deleted the resources (if necessary), build them back up (or for
 	// the first time if they're new).
-	if tpNeedsUpdate {
+	if tpNeedsRecreation {
 		createInstances := hosts
```

Comment:
split this out to its own function...

Reply:
Ack, split to a new function in the second commit. No functional change.